### PR TITLE
Dop 2366: Add Unified Footer

### PR DESCRIPTION
**Pull request by casthewiz**
### Stories/Links:

DOP-2366

### Staging Links:

[Realm](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/cassidy.schaufele/DOP-2366/)
[Manual](https://docs-mongodbcom-staging.corp.mongodb.com/master/docs/cassidy.schaufele/DOP-2366/)

### Notes:
- Adds the UnifiedFooter to docs properties, with the existing feature flag
- Makes both the sidenav and ToC `sticky` instead of `fixed` to resolve overlap issues with the new tall footer.
- Ridealong formatting of CHANGELOG.md

### CODE REVIEW

File: src/components/RightColumn.js [hunk 1]
```diff
@@ -18,8 +18,9 @@ const RightColumn = ({ children }) => (
       css={css`
         height: 100%;
         max-height: calc(100vh - 120px);
-        overflow: auto;
-        position: fixed;
+        overflow: scroll;
+        position: sticky;
+        top: 99px;
```
Comment (casthewiz) [inline src/components/RightColumn.js:23]: `top: 99px` allows us to maintain a visual 12px margin above and below the "On this page" header within the ToC

**Review by allisonmui9 — COMMENTED** [review]
everything LGTM! besides the hover state on the links. Is that how it was designed?

**Comment by casthewiz** [comment]
> everything LGTM! besides the hover state on the links. Is that how it was designed?

Hmmm - looks like we're applying some styling on them from our legacy css. I'll look at mitigating 🤔

**Review by mango-db — APPROVED** [review]
LGTM: [Node staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/node/cgoecknerwald/DOP-2366/). [Realm staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/cgoecknerwald/DOP-2366/). [Drivers staging (no flag)](https://docs-mongodbcom-staging.corp.mongodb.com/master/drivers/cgoecknerwald/DOP-2366/)

- [ ] Consistent nav dropdown don't seem to work on Firefox, but it doesn't seem to be related to the work in this PR
- [x] But otherwise this ticket LGTM

**Pull request merged**